test/uk/gov/hmrc/helptosave/connectors/ApiTwentyFiveCConnectorSpec.scala

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosave.connectors
+
+import org.joda.time.LocalDate
+import org.scalatest._
+import org.scalamock.scalatest.MockFactory
+import play.api.libs.json.Json
+import uk.gov.hmrc.helptosave.config.WSHttp
+import uk.gov.hmrc.helptosave.models.{ApiTwentyFiveCValues, AwAwardStatus, Award}
+import uk.gov.hmrc.helptosave.util.NINO
+import uk.gov.hmrc.play.http.{HeaderCarrier, HttpResponse}
+import uk.gov.hmrc.play.test.WithFakeApplication
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.concurrent.duration._
+
+class ApiTwentyFiveCConnectorSpec extends WordSpec with WithFakeApplication with Matchers with MockFactory {
```

Comment:
Lets define a TestSupport class as we have for frontend `https://github.com/hmrc/help-to-save-frontend/blob/master/test/uk/gov/hmrc/helptosavefrontend/TestSupport.scala` and extend it for all specs ?